routes/fractie.ts

```diff
@@ -9,12 +9,30 @@ import { mandatarisUsecase } from '../controllers/mandataris';
 export const fractiesRouter = Router();
 
 fractiesRouter.get(
-  '/:bestuursperiodeId/bestuursperiode',
+  '/onafhankelijk/:bestuursperiodeId/bestuursperiode',
   async (req: Request, res: Response) => {
     const id = req.params.bestuursperiodeId;
 
     try {
-      const fractieIds = await fractieUsecase.forBestuursperiode(id);
+      const fractieIds = await fractieUsecase.forBestuursperiode(id, true);
+      return res.status(STATUS_CODE.OK).send({ fracties: fractieIds });
+    } catch (error) {
+      const message =
+        error.message ??
+        `Something went wrong while getting fracties for bestuursperiod: ${id}`;
+      const statusCode = error.status ?? STATUS_CODE.INTERNAL_SERVER_ERROR;
+      return res.status(statusCode).send({ message });
+    }
+  },
+);
+
+fractiesRouter.get(
+  '/samenwerking/:bestuursperiodeId/bestuursperiode',
```

Comment:
Well, I'm a bit conflicted about it. I don't think we need it and it feels abundant indeed, but otherwise, it feels less safe or clear if I do it with a boolean or something like that. I can check if I can remove some duplication.